islandora_video_streaming_kaltura.module

```diff
@@ -60,7 +59,7 @@ dpm('registering deriviative');
  *
  * @var $variables array The page content variables
  */
-function islandora_video_streaming_kaltura_process_islandora_video($variables = array()) {
+function islandora_video_streaming_kaltura_process_islandora_video(&$variables = array()) {
```

Comment:
This should have been part of a separate pull request, but we can leave it in here for convenience.